packages/eslint-plugin-office-addins/test/rules/call-sync-after-load.test.ts

```diff
@@ -75,5 +82,12 @@ ruleTester.run('call-sync-after-load', rule, {
 			  console.log(range.font.fill.color);`,
```

Comment:
There is something wrong in the way vscode vs github see indentation, because I see the code normally on my editor:
![image](https://user-images.githubusercontent.com/18221895/135539708-2df67662-5384-4985-8111-ec1284fc2195.png)

Something to look out for, I will see if github detects any other indentation error as well.

Reply:
It's likely a difference it tabs vs. spaces . . . that is usually what triggers a miss match since tab get interpreted differently depending on what is opening the file..